<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transceiverAdjustments</class>
 <widget class="QWidget" name="transceiverAdjustments">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QGroupBox" name="transmitterControlsGroupBox">
       <property name="title">
        <string>Transmitter</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="compVertLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSlider" name="compSlider">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>230</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="compChkBox">
            <property name="text">
             <string>Comp</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="bassTxVertLayout">
          <item>
           <widget class="QSlider" name="bassTxSlider">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>230</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="bassTxLabel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>Bass</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="trebleTxVertLayout">
          <item>
           <widget class="QSlider" name="trebleTxSlider">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>230</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="trebleTxLabel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>Treble</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="tbdVertLayout">
          <item>
           <widget class="QSlider" name="tbdSlider">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>230</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="tbdLabel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>TBD</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="receiverControlsGroupBox">
       <property name="title">
        <string>Receiver Passband Controls</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="PBTOuterLabel">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>PBF Outer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="PBTOuterSlider">
            <property name="minimumSize">
             <size>
              <width>230</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="PBTInnerLabel">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>PBF Inner</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="PBTInnerSlider">
            <property name="minimumSize">
             <size>
              <width>230</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="IFShiftLabel">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>IF Shift</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="IFShiftSlider">
            <property name="minimumSize">
             <size>
              <width>230</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Filter Width</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="passbandWidthSlider">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QPushButton" name="resetPBTbtn">
            <property name="text">
             <string>Reset PBT</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="otherGrpBox">
       <property name="title">
        <string>Other</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="bassRxVertLayout">
          <item>
           <widget class="QSlider" name="bassRxSlider">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>230</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="bassRxLabel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>Bass</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="trebleRxVertLayout">
          <item>
           <widget class="QSlider" name="trebleRxSlider">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>230</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="trebleRxLabel">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>Treble</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="NRRxVertLayout">
          <item>
           <widget class="QSlider" name="NRRxSlider">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>230</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="NRRxCheckBox">
            <property name="text">
             <string>NR</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="NBRxVertLayout">
          <item>
           <widget class="QSlider" name="NBRxSlider">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>230</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="NBRxChkBox">
            <property name="text">
             <string>NB</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="notchRxVertLayout">
          <item>
           <widget class="QSlider" name="notchRxSlider">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>230</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="notchRxChkBox">
            <property name="text">
             <string>Notch</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="bandwidthGroupBox">
          <property name="title">
           <string>Bandwidth</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_12">
           <item>
            <widget class="QLabel" name="lowBandwidthLabel">
             <property name="text">
              <string>Low</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="lowBandwidthCombo"/>
           </item>
           <item>
            <widget class="QLabel" name="highBandwidthLabel">
             <property name="text">
              <string>High</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="highBandwidthCombo"/>
           </item>
           <item>
            <widget class="QLabel" name="filterBandwidthLabel">
             <property name="text">
              <string>Filter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="filterBandwidthCombo"/>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
